src/open_inwoner/utils/validators.py

```diff
@@ -1,12 +1,75 @@
+from typing import TYPE_CHECKING, Protocol
+
 from django.core.exceptions import ValidationError
 from django.core.files.images import get_image_dimensions
 from django.core.validators import RegexValidator
 from django.utils.deconstruct import deconstructible
 from django.utils.encoding import force_text
 from django.utils.translation import gettext_lazy as _
 
+import phonenumbers
 from filer.models import Image
 
+if TYPE_CHECKING:
+    from phonenumbers.phonenumber import PhoneNumber
+
+
+class ParsePhoneNumber(Protocol):
+    def __call__(self, value: str) -> "PhoneNumber":
+        ...  # pragma: nocover
+
+
+@deconstructible
+class DutchPhoneNumberValidator:
+    country = "NL"
+    country_name = _("Dutch")
```

Comment:
"Dutch" is the name of the language. Either change the label to "language" or the field to _("Netherlands")